src/utils/cloning.ts

```diff
@@ -391,6 +394,27 @@ function cloneASTNodeImpl<T extends ASTNode>(
       node.sourceUnit,
       node.raw,
     );
+  } else if (node instanceof StructDefinition) {
```

Comment:
Each group in cloning.ts is meant to be in alphabetical order for easier traversal (though I notice ImportDirective was not following that convention)